docs/continuing_runs.md

```diff
@@ -0,0 +1,225 @@
+
+Let's imagine you execute a run with six steps (modules).
+
+```
+haddock3 my-haddock3-config-file.cfg
+```
+
+If the run completes successfully, the run directory shows as follows:
+
+```
+0_topoaa/
+1_rigidbody/
+2_caprieval/
+3_seletop/
+4_flexref/
+5_caprieval/
+data
+log
+```
+
+Now, imagine that later in your analysis, you notice that you used the
+wrong parameters for the flexible refinement (`4_flexref`) stage,
+setting `randremoval=true` instead of `randremoval=false`. Therefore,
+you want to repeat the run only for `4_flexref` onwards without
+repeating the previous successful steps. For that, edit the
+configuration file according to your new preferences, and run:
+
+```
+haddock3 my-haddock3-config-file.cfg --restart 4
+```
+
+This operation will delete modules' folders `4_` onwards and repeat
+those steps following the configurations in the (new) configuration
+file. You must run the `haddock3` command from within the same directory
+as you did for the original run.
+
+As you can see, this option allows you to **restart** or **repeat** a
+run from a specific step using the **original** configuration file or to
+**modify** a run by updating the parameters for the steps you want to
+rerun.
+
+Likewise, we can use the `--restart` functionality to **extend** a run
+with additional steps. For example, Imagine `run 1` completed
+successfully. Still, you now want to complement it with an `emref` step,
+perform a final CAPRI score evaluation, and finalize the run by
+clustering with FCC (`clustfcc` module). For that, add the parameters
+for the new modules to the end of the original configuration file. The
+original steps' order in the configuration file must be kept the same
+for HADDOCK3 to understand *the previous* with the *new* modules.
+
+Navigate back to the same folder you executed the previous runs and:
+
+```
+haddock3 expanded-configuration-file.cfg --restart 6
+```
+
+In this case, the `--restart` option takes the value `6` because we want to
+continue the run from step 6, the new `emref` step. This would result in:
+
+```
+0_topoaa/
+1_rigidbody/
+2_caprieval/
+3_seletop/
+4_flexref/
+5_caprieval/
+6_emref/
+7_caprieval/
+8_clustfcc/
+data
+log
+```
+
+As you can see, you can use `--restart` to edit and rerun modules, to
+add new modules, **or do both at the same time**.
+
+# Copy a run to a new folder, and continue from there
+
+You can execute all previous examples on copies of the original run.
+There is no need to rewrite the initial run if you wish to keep it. For
+that, copy the original run to a new folder:
+
+```
+cp -r run1 run2
+```
+
+You can also copy the run to different folders because HADDOCK3 treats
+internal paths as relative paths. Therefore, modules can communicate
+with each other regardless of the absolute location of the run
+directory. The same applies to the installation requirements. Provided
+the installation process was the same, you can copy HADDOCK3 run over
+different machines. 
```

Comment:
Are you sure about this one? I've tried it once with a previous version - maybe its fixed now, from my workstation to the cluster and it did not work.

Reply:
It should work now, provided the haddock3 environment is installed. all haddock3 libraries are taken from the system installation. The run does not care about where things are installed, as long as they are accessible in the system, which they will be if installed following the INSTALL.md instructions.